test/WebJobs.Script.Tests/PowerShellEndToEndTests.cs

```diff
@@ -46,7 +46,7 @@ public PowerShellEndToEndTests(TestFixture fixture)
 
             HttpRequestMessage request = new HttpRequestMessage
             {
-                RequestUri = new Uri("http://localhost/api/httptrigger-powershell?code=1388a6b0d05eca2237f10e4a4641260b0a08f3a5&name=testuser"),
```

Comment:
I just cleaned up the PS sample functions - they didn't need to have Powershell in their names, since they're in a Powershell folder. Same pattern as all other tests.